modules/migration-installing-mtc-on-ocp-3.adoc

```diff
@@ -98,14 +87,14 @@ containers:
     - name: REGISTRY
       value: <registry.apps.example.com> <3>
 ----
-<1> Specify your mirror registry and the `sha256` value of the Operator image in the `mapping.txt` file.
-<2> Specify your mirror registry and the `sha256` value of the Operator image in the `mapping.txt` file.
+<1> Specify your mirror registry and the `sha256` value of the Operator image.
```

Comment:
I know it was already like this, but same comment about using the same callout number for multiple places